### ISSUE TITLE
[BUG] Missing hinge/rotator/rail info on mouse-over

### ISSUE THREAD

**Issue by alxnns1** [tsunamayo/Starship-EVO#2458]
### Short Description of the issue:

The debug info that used to show while mousing over a hinge, rotator or rail is missing. It use to show the current angle/position of the block along with some other useful bits.

### Environment

- Starship EVO version: 20w25d
- Graphics Card: GTX 1050 Ti

### Additional information:

![20200619200835_1](https://user-images.githubusercontent.com/485571/85172648-6b50f780-b269-11ea-8c63-89f401e9b703.jpg)

[output_log.txt](https://github.com/tsunamayo/Starship-EVO/files/4806363/output_log.txt)

**Comment by tsunamayo** [comment]
Yes I am still migrating this! Will come back shortly, except you will have to use the link tool to select the system.
